src/components/nav-menu-item/menu-item.tsx

```diff
@@ -77,15 +77,15 @@ export class CalciteNavMenuItem implements LoadableComponent {
   @Prop({ reflect: true }) target: string;
 
   /** Specifies the text the component displays */
-  @Prop({ reflect: true, mutable: true }) text: string;
+  @Prop({ reflect: true }) text: string;
 
   /** Displays the `text` */
-  @Prop({ mutable: true }) textEnabled: boolean;
+  @Prop() textEnabled: boolean;
 
   /**
    * @internal
    */
-  @Prop({ mutable: true }) subMenuOpen = false;
+  @Prop({ mutable: true }) open = false;
```

Comment:
I know this is internal - 2 questions - is `open` descriptive enough? And, should this be publicly documented (is there a reason a dev user might want to be able to toggle this? I could see it maybe being useful in `layout="vertical"` menu use cases.

Reply:
i gave a thought about changing it to `open`, the reason being consistency. Though it is internal, keeping it consistent will be useful if we choose to make it public in future as you mentioned for `layout="vertical"`